solveengine/basemodel.py

```diff
@@ -65,18 +70,23 @@ class BaseModel(object):
     BASEURL = "https://solve.satalia.com/api/v2/jobs/"
```

Comment:
URLs should not be hardcoded into codebase. They should be extracted in configuration file and used from there. Same is with results, status and schedule urls. It does not matter is this experimenting, researching and etc. Good programming practice should be uses.